### PR TITLE
Coding week 2 and 3

### PR DESCRIPTION
**Pull request by RAJAGOPALAN-GANGADHARAN**

### CODE REVIEW

File: content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* Phew and finally getting the processes to get to know about each other (duh by making them talk).
+
+<h1>Living in the Present</h1>
+Ah yes after completing these I feel totally awesome and stepped up the game, And that brings us to the latest progress....
+
+Now the main feature of a browser is to show webpages. hell no!! It has got other cool stuff too.
+
+So first I did easy things like showing the status of the progress, keeping track of history( to go back and forward 🚗). For this I had to complete implementing Shared memory :heavenly tones play: . Sounds cool eh? Yes Shared memory is about allocating memory and sharing it with other processes ,this is done to keep track of visited processes.
+
+(Psst [click-here](https://imgur.com/a/0Ny2sj5) for some cool stuff)
```
Comment (pulkomandy) [inline content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md]: Please add the images to our git repo instead, we don't know if imgur is going to keep them long enough (we have some 15 years old articles where images went missing and it's annoying)

File: content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Oh ya not to mention >:making friends!:<
+* Taming git and messing up with the repo.
+* Phew and finally getting the processes to get to know about each other (duh by making them talk).
+
+<h1>Living in the Present</h1>
+Ah yes after completing these I feel totally awesome and stepped up the game, And that brings us to the latest progress....
+
+Now the main feature of a browser is to show webpages. hell no!! It has got other cool stuff too.
+
+So first I did easy things like showing the status of the progress, keeping track of history( to go back and forward 🚗). For this I had to complete implementing Shared memory :heavenly tones play: . Sounds cool eh? Yes Shared memory is about allocating memory and sharing it with other processes ,this is done to keep track of visited processes.
```
Comment (pulkomandy) [inline content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md]: space after , not before

File: content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+(Psst [click-here](https://imgur.com/a/0Ny2sj5) for some cool stuff)
+
+Yeah that's right we have loaded a blank url completely, I bet your mind is blown 🤯.
+
+Oh ya totally forgot about the cool debugging system we use. We use BeDC to get outputs from different process in cool colours(its magical trust me or take a look [here](https://imgur.com/a/YSWQBSz))
```
Comment (pulkomandy) [inline content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md]: You could include a link to the BeDC archive (http://pulkomandy.tk/drop/BeDC-1.0.zip)

File: content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Now before we get to rendering we need to get the webpage of source cause nobody is ever going to load about:blank. duh!
+So we started working on NetworkProcess and hey I managed to make Http requests using BHttpRequest and BUrlRequest and got the html in bits and pieces. Few parts from the well maintained WebKitLegacy port were stolen and used in NetworkProcess. So we are doing some more work to make it "usable". All pray for it to go really smooth 🤞🏻.
+
+<h1>Hey look its already Future</h1>
```
Comment (pulkomandy) [inline content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md]: h1 is reserved for the article title. Usually we start article content at the h3 level instead

File: content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md (hunk 1 above)
Comment (RAJAGOPALAN-GANGADHARAN) [reply content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md]: Last time the image was just gone, maybe I followed wrong procedure?
Whats the correct one to add image?

**Comment by pulkomandy** [comment]
git add to add it to the repo. If it disappears, then it's not your problem, the website team should fix it.

**Pull request merged**